Gemfile

```diff
@@ -6,6 +6,9 @@ ruby "2.7.2"
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem "rails", "6.1.3.1"
 
+# Transpile app-like JavaScript. Read more: https://github.com/rails/webpacker
+gem "webpacker"
```

Comment:
might be worth adding the version here, the latest pre-release 6.x in beta introduces some breaking changes from the current stable 5.x release

Reply:
Fixed